packages/button/src/button.tsx

```diff
@@ -147,7 +147,7 @@ export const Button = forwardRef<ButtonProps, "button">((props, ref) => {
       )}
 
       {isLoading ? (
-        loadingText || (
+        (loadingText &&  <chakra.span>{loadingText}</chakra.span>) || (
```

Comment:
Could you run prettier on this file? Otherwise this will be marked as changed when the next person formats this file.

Reply:
@TimKolberger 
Has been updated.